openqabot/types/incidents.py

```diff
@@ -243,9 +256,14 @@ def _handle_incident(  # pylint: disable=too-many-return-statements
         if "params_expand" in data:
             full_post["openqa"].update(data["params_expand"])
 
-        full_post["openqa"][
-            "__SMELT_INCIDENT_URL"
-        ] = f"https://smelt.suse.de/incident/{inc.id}"
+        if inc.project == "SLFO":
+            full_post["openqa"][
+                "__GITEA_PR_URL"
+            ] = f"{GITEA}/products/{inc.project}/pulls/{inc.id}"
+        else:
+            full_post["openqa"][
+                "__SMELT_INCIDENT_URL"
+            ] = f"https://smelt.suse.de/incident/{inc.id}"
```

Comment:
```suggestion
            ] = f"{SMELT}/incident/{inc.id}"
```

We could get rid of this hardcoded value, so `SMELT_URL` will apply.

Reply:
Ok - although I don't think I have introduced this line :-)